src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -51,46 +51,63 @@ abstract class LeoRoutes(val leonardoService: LeonardoService, val proxyService:
             }
           }
         } ~
-        pathPrefix("cluster" / Segment / Segment) { (googleProject, clusterName) =>
-          pathEndOrSingleSlash {
-            put {
-              entity(as[ClusterRequest]) { cluster =>
-                complete {
-                  leonardoService.createCluster(userInfo, GoogleProject(googleProject), ClusterName(clusterName), cluster).map { cluster =>
-                    StatusCodes.OK -> cluster
+        pathPrefix("cluster") {
+          pathPrefix("v2" / Segment / Segment) { (googleProject, clusterName) =>
+            pathEndOrSingleSlash {
+              put {
+                entity(as[ClusterRequest]) { cluster =>
+                  complete {
+                    leonardoService
+                      .syncCreateCluster(userInfo, GoogleProject(googleProject), ClusterName(clusterName), cluster)
```

Comment:
Shouldn't this be asyncCreateCluster?

Reply:
+1, I think asyncCreateCluster would be a clearer name because this variant creates a Cluster asynchronously.

Reply:
I think I see, `syncCreateCluster` is for the sync portion of the creation and `asyncClusterCreate` is for the async portion. It's a bit confusing because `leonardoService.syncCreateCluster` is _async_ compared with `leonardoService.createCluster`. :) Maybe there's some naming scheme for these methods so the intention is more clear. LeonardoService is not public-facing so we should be free to refactor names, etc.